versioned_docs/version-8.2/components/modeler/forms/form-element-library/forms-element-library-image.md

```diff
@@ -10,7 +10,7 @@ An element allowing the user to display images.
 
 ## Configurable properties
 
-- **Image source**: Specifies the image source via [expression](../../feel/language-guide/feel-expressions-introduction.md) or static value (hyperlink or data URI).
+- **Image source**: Specifies the image source via [expression](../../feel/language-guide/feel-expressions-introduction.md) or [static value](/docs/components/concepts/expressions.md) (hyperlink or data URI).
```

Comment:
🤔 I'm not familiar with this part, I wonder whether this static value is as defined by the concepts. I hope someone else can comment on this one.

Reply:
@Skaiir Can you validate this is a solid addition?